scene/gui/graph_edit.cpp

```diff
@@ -1385,4 +1406,12 @@ GraphEdit::GraphEdit() {
 	setting_scroll_ofs = false;
 	just_disconnected = false;
 	set_clip_contents(true);
+	theme = get_theme();
```

Comment:
Don't add them by default, let the user add them if needed. By default it should have no zoom levels and zoom controls should not be visible.

Reply:
Can you explain this in another way? The user? The game engine developer?